packages/mobile/src/redux/store.ts

```diff
@@ -129,21 +132,31 @@ export const configureStore = (initialState = {}) => {
     )
   }
 
-  const enhancers = [applyMiddleware(...middlewares)]
+  middlewares.push(cloudFunctionsApi.middleware)
+
+  const enhancers = []
 
   if (__DEV__) {
     const Reactotron = require('src/reactotronConfig').default
     enhancers.push(Reactotron.createEnhancer())
   }
 
-  const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose
-  // @ts-ignore
-  const createdStore = createStore(persistedReducer, initialState, composeEnhancers(...enhancers))
-
```

Comment:
Do we keep redux devtools observability in Flipper with this change? is it handled in `configureStore` from Redux ToolKit?

Reply:
All good, I just checked and it does it in RTK: https://github.com/reduxjs/redux-toolkit/blob/master/packages/toolkit/src/devtoolsExtension.ts